persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_26/generated/entities/generated_client.bal

```diff
@@ -7,8 +7,8 @@ import ballerina/persist;
 import ballerina/sql;
 import ballerinax/mysql;
 
-const MEDICALNEED = "MedicalNeed";
-const MEDICALITEM = "MedicalItem";
+const MEDICAL_NEED = "MedicalNeed";
+const MEDICAL_ITEM = "MedicalItem";
```

Comment:
```suggestion
const MEDICAL_ITEM = "medicalitem";
```

Reply:
value should be lowercase right?

Reply:
updated in [5b7ba2a](https://github.com/ballerina-platform/persist-tools/pull/111/commits/5b7ba2aee172309921203f9104d2ce537a0fbab2)